database/seeders/samples/TestDataSeeder.php

```diff
@@ -0,0 +1,401 @@
+<?php
+
+namespace Database\Seeders\Samples;
+
+use App\Models\AffiliateSite;
+use App\Models\Answer;
+use App\Models\Genre;
+use App\Models\Interest;
+use App\Models\Keyword;
+use App\Models\Place;
+use App\Models\PlaceAffiliateSite;
+use App\Models\PlaceFavorite;
+use App\Models\PlaceImage;
+use App\Models\PlaceKeyword;
+use App\Models\Plan;
+use App\Models\PlanDetail;
+use App\Models\PlanFavorite;
+use App\Models\PlanKeyword;
+use App\Models\Question;
+use App\Models\User;
+use Illuminate\Database\Seeder;
+
+class TestDataSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        // #Users
+        $f = fopen("database/seeders/samples/csv/users.csv", "r");
+
+        $cnt = 1;
+        while($line = fgetcsv($f)) {
+            if ($cnt !== 1) {
+                User::create([
+                    'id'  => $line[0],
```

Comment:
we should not use an id from third party data. we should let our Model create the value of the id

Reply:
Please do the same with other test data. Do not use the id from the csv file